runtime/doc/intro.txt

```diff
@@ -237,73 +237,74 @@ These names for keys are used in the documentation.  They can also be used
+
+<Up>            Cursor-up                       *cursor-up* *cursor_up*
+<Down>          Cursor-down                     *cursor-down* *cursor_down*
+<Left>          Cursor-left                     *cursor-left* *cursor_left*
+<Right>         Cursor-right                    *cursor-right* *cursor_right*
+<S-Up>          Shift-cursor-up
+<S-Down>        Shift-cursor-down
+<S-Left>        Shift-cursor-left
+<S-Right>       Shift-cursor-right
+<C-Left>        Control-cursor-left
+<C-Right>       Control-cursor-right
+<F1> - <F12>    Function keys 1 to 12           *function_key* *function-key*
+<S-F1> - <S-F12> Shift-function keys 1 to 12    *<S-F1>*
+<Help>          Help key
+<Undo>          Undo key
+<Find>          Find key
+<Select>        Select key
+<Insert>        Insert key
+<Home>          Home                            *home*
+<End>           End                             *end*
+<PageUp>        Page-up                         *page_up* *page-up*
+<PageDown>      Page-down                       *page_down* *page-down*
+<kUp>           Keypad cursor-up                *keypad-cursor-up*
+<kDown>         Keypad cursor-down              *keypad-cursor-down*
+<kLeft>         Keypad cursor-left              *keypad-cursor-left*
+<kRight>        Keypad cursor-right             *keypad-cursor-right*
+<kHome>         Keypad home (upper left)        *keypad-home*
+<kEnd>          Keypad end (lower left)         *keypad-end*
+<kOrigin>       Keypad origin (middle)          *keypad-origin*
+<kPageUp>       Keypad page-up (upper right)    *keypad-page-up*
+<kPageDown>     Keypad page-down (lower right)  *keypad-page-down*
+<kDel>          Keypad delete                   *keypad-delete*
+<kPlus>         Keypad +                        *keypad-plus*
+<kMinus>        Keypad -                        *keypad-minus*
+<kMultiply>     Keypad *                        *keypad-multiply*
+<kDivide>       Keypad /                        *keypad-divide*
+<kPoint>        Keypad .                        *keypad-point*
+<kComma>        Keypad ,                        *keypad-comma*
+<kEqual>        Keypad =                        *keypad-equal*
+<kEnter>        Keypad Enter                    *keypad-enter*
+<k0> - <k9>     Keypad 0 to 9                   *keypad-0* *keypad-9*
 <S-…>           shift-key                       *shift* *<S-*
-<C-…>           control-key                     *control* *ctrl* *<C-*
-<M-…>           alt-key or meta-key             *META* *ALT* *<M-*
-<A-…>           same as <M-…>                   *<A-*
-<T-…>           meta-key when it's not alt      *<T-*
-<D-…>           command-key or "super" key      *<D-*
+<C-…>           Control-key                     *control* *ctrl* *<C-*
+<M-…>           Alt-key or meta-key             *META* *ALT* *<M-*
+<A-…>           Same as <M-…>                   *<A-*
+<T-…>           Meta-key, when it's not alt     *<T-*
+<D-…>           Command-key or "super" key      *<D-*
```

Comment:
The capitalization here is inconsistent.

Reply:
I missed the `shift-key` 🤦